samples/cellular/nrf_cloud_multi_service/boards/thingy91x_nrf9151_ns.conf

```diff
@@ -56,19 +55,19 @@ CONFIG_NET_IF_MAX_IPV4_COUNT=2
 CONFIG_NET_IF_MAX_IPV6_COUNT=2
 
 # Memory configurations
-CONFIG_NET_BUF_RX_COUNT=8
-CONFIG_NET_BUF_TX_COUNT=8
+CONFIG_NET_BUF_RX_COUNT=1
+CONFIG_NET_BUF_TX_COUNT=1
 CONFIG_NET_PKT_RX_COUNT=1
 CONFIG_NET_PKT_TX_COUNT=1
-CONFIG_NET_TX_STACK_SIZE=4096
-CONFIG_NET_RX_STACK_SIZE=4096
+CONFIG_NET_TX_STACK_SIZE=512
+CONFIG_NET_RX_STACK_SIZE=512
 CONFIG_NET_TC_TX_COUNT=1
-CONFIG_NET_MAX_CONTEXTS=5
+CONFIG_NET_MAX_CONTEXTS=1
```

Comment:
@maxd-nordic I wonder if this conflicts with `CONFIG_NET_IF_MAX_IPV4_COUNT=2` and `CONFIG_NET_IF_MAX_IPV6_COUNT=2`? We found that these needed to be 2 and not 1 when using Wi-Fi scanning.

Reply:
Although I haven't seen any failures with MSS on my devices (thingy91x and DK) so far, I have a bit of doubts about these optimizations because MSS is using LTE net_if and might hence use NET stacks and resources differently to other applications/samples.

Reply:
Still, everything is working fine for me but I don't know if there is a configuration (SIM, network) where this won't work.
I reverted this just to be safe. Changing between 5 and 1 is 772 bytes of RAM.

Reply:
@plskeggs You are likely on Thanksgiving holidays. I'm requesting this to get merged so we get feedback from the wider CI systems as well. If you have any comments, please post them still to the PR and I'll make follow up PRs but at least we get the major changes into main.

Reply:
Ok, seems this cannot be merged without your approval from @nrfconnect/ncs-nrf-cloud

Reply:
Sorry. Approved now.